R/step_growth_rate.R

```diff
@@ -46,11 +49,11 @@ step_growth_rate <-
            horizon = 7,
            method = c("rel_change", "linear_reg"),
            log_scale = FALSE,
+           na_rm = TRUE,
            replace_Inf = NA,
            prefix = "gr_",
            skip = FALSE,
-           id = rand_id("growth_rate"),
-           additional_gr_args_list = list()) {
```

Comment:
I'm wondering why you're dropping `additional_gr_args_list`; `growth_rate` appears to still have a `params` option. Were you planning on not passing those through?

Reply:
Currently, the only supported methods are `rel_change` and `linear_reg`. The others can't be used for forecasting (without major changes to lots of things). Those methods don't need/allow any additional arguments through that list.

Reply:
I had local tests passing until I opened an unrelated PR. Locally, I had epiprocess 0.10.0. But if you're on current epiprocess dev (>=0.10.4) then the tests all fail. (and they also fail on CI after it installs from the default branch)

Reply:
right, makes sense. Thought I had local tests passing on epiprocess 0.10.4, but I must've just been confused. Thanks for the clarification.